libraries/AP_Scripting/docs/docs.lua

```diff
@@ -4013,3 +4013,24 @@ function visual_odom:healthy() end
 -- visual odometry quality as a percentage from 1 to 100 or 0 if unknown
 ---@return integer
 function visual_odom:quality() end
+
+-- Returns the number of detected obstacles via ADS-B
+-- the ADS-B Must be enabled.
+---@return integer -- number of obstacles
+function avoid:num_obstacles() end-- Returns a Location userdata for the last position of the obstacle observed.
+
+---@param instance integer -- instance number
+---@return Location_ud --ADSB location
+function avoid:get_obstacle_loc(instance) end-- Returns a Vector3f that contains the velocity as obtained from ADS-B.
+
+---@param instance integer -- instance number
+---@return Vector3f_ud -- 3D velocity in m/s, in NED format
+function avoid:get_obstacle_vel(instance) end-- Obstacle id as obtained from ADS-B.
```

Comment:
Can't this be other obstacles? Not just ADS-B? What about Remote ID vehicles or something identified by an off board AI/camera?